### PR TITLE
Add demo instructions on WebUI

### LINKED ISSUE #339: Try Cartridge: default state button
We need to implement button for restoring demo mode to default state

### PR DESCRIPTION
**Pull request by KlonD90**
- Add Demo Uri Information and "how to connect Info" to Code Page
- Add Reset button for demo purposes to Code Page

I didn't forget about

- [x] Tests
- [x] Changelog
- [x] Documentation (not applicable)

Close #339

### CODE REVIEW

File: webui/src/components/DemoInfo.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+import { connect } from 'react-redux'
+import type { State } from '../store/rootReducer'
+import { Panel } from './Panel'
+import { css } from 'emotion'
+import styled from 'react-emotion'
+import {
+  ConfirmModal, PopupBody, Text, Modal, Tabbed, PopupFooter, Button, Markdown
+} from '@tarantool.io/ui-kit'
+
+const uriDecompose = uri => {
```
Comment (rgbabaev) [inline webui/src/components/DemoInfo.js]: It crashes in cases when uri don't have credentials

File: webui/src/components/DemoInfo.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            </p>
+            <p>
+              <Text>This action will result in data loss.</Text>
+            </p>
+          </PopupBody>
+        </ConfirmModal>
+      }
+      {
+        isShowConnectInfo &&
+        <Modal visible={isShowConnectInfo} title={'Connect info'} onClose={this.hideConnectInfo}>
```
Comment (rgbabaev) [inline webui/src/components/DemoInfo.js]: This Modal will look prettier with `wide={true}` prop

File: webui/src/components/DemoInfo.js [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+\`\`\`
+`,
+    decomposed: true
+  },
+  PHP: {
+    markdown: `
+## Connect to Tarantool Cartridge using [php client](https://github.com/tarantool-php/client)
+
+First, **install** *messagepack* and *tarantool client* using *composer*:
+\`\`\`bash
```
Comment (rgbabaev) [inline webui/src/components/DemoInfo.js]: Add one more new line before code block for right formatting in markdown-to-jsx

File: webui/src/components/DemoInfo.js [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+$space = $client->getSpace('customer');
+$space->insert([222, 'Michael Bryan']);
+$result = $space->select(Criteria::index(0));
+
+print_r($result);
+?>
+\`\`\`
+
+**Run** the script  
+\`\`\`bash
```
Comment (rgbabaev) [inline webui/src/components/DemoInfo.js]: New line

**Comment by artur-barsegyan** [comment]
Commit messages are misleading. Need to change it and squash.

**Pull request merged**